src/map/mob_spell_container.cpp

```diff
@@ -60,7 +60,12 @@ void CMobSpellContainer::AddSpell(SpellID spellId)
         m_gaList.push_back(spellId);
 
     }
-    else if(spell->canTargetEnemy()){
+    else if(spell->isSevere()){
+        // select spells like death and impact
+        m_severeList.push_back(spellId);
+        
```

Comment:
got an extra blank line here

Reply:
After dialogue: fixed only for the affected lines by this PR for cleanliness of the PR itself. See ff24165b3b3b2dcc6f2b07f85ffad18954bbdbdb